library/std/canon.qs

```diff
@@ -125,9 +125,14 @@ namespace Microsoft.Quantum.Canon {
     /// # Summary
     /// Applies the controlled-X (CX) gate to a pair of qubits.
     ///
-    /// # Description
+    /// # Input
+    /// ## control
+    /// Control qubit for the CX gate.
+    /// ## target
+    /// Target qubit for the CX gate.
+    ///
+    /// # Remarks
     /// This operation can be simulated by the unitary matrix
-    /// $$
```

Comment:
Is there a reason to remove the LaTex delimiters? How would automated and parsing tools be able to know where LaTex begins and ends?

Reply:
For consistency and expressiveness, I would suggest enclosing LaTex between `$` and `$$` delimiters even if the tools that we have used so far do not require it.

For search purposes, if you want to look at where LaTex is used across the codebase, it's easier to look for the delimiters than to look for the delimiters and `\begin`.

Reply:
I've put the `$$` back in the canon.qs and for consistency added them to intrinsic.qs as well.